fusestore/store/src/api/rpc/flight_service.rs

```diff
@@ -5,39 +5,56 @@
 use std::convert::TryInto;
 use std::pin::Pin;
 
-use common_arrow::arrow_flight::{
-    self,
-    flight_service_server::{FlightService as Flight, FlightServiceServer as FlightServer},
-    Action, ActionType, BasicAuth, Criteria, Empty, FlightData, FlightDescriptor, FlightInfo,
-    HandshakeRequest, HandshakeResponse, PutResult, SchemaResult, Ticket,
-};
+use common_arrow::arrow_flight;
+use common_arrow::arrow_flight::flight_service_server::FlightService;
+use common_arrow::arrow_flight::flight_service_server::FlightServiceServer;
+use common_arrow::arrow_flight::Action;
+use common_arrow::arrow_flight::ActionType;
+use common_arrow::arrow_flight::BasicAuth;
+use common_arrow::arrow_flight::Criteria;
+use common_arrow::arrow_flight::Empty;
+use common_arrow::arrow_flight::FlightData;
+use common_arrow::arrow_flight::FlightDescriptor;
+use common_arrow::arrow_flight::FlightInfo;
+use common_arrow::arrow_flight::HandshakeRequest;
+use common_arrow::arrow_flight::HandshakeResponse;
+use common_arrow::arrow_flight::PutResult;
+use common_arrow::arrow_flight::SchemaResult;
+use common_arrow::arrow_flight::Ticket;
 use common_flights::store_do_action::StoreDoAction;
 use common_flights::store_do_get::StoreDoGet;
 use common_flights::{FlightClaim, FlightToken};
-use futures::{Stream, StreamExt};
+use futures::Stream;
+use futures::StreamExt;
+#[allow(unused_imports)]
+use log::error;
+#[allow(unused_imports)]
 use log::info;
 use prost::Message;
 use tonic::metadata::MetadataMap;
-use tonic::{Request, Response, Status, Streaming};
+use tonic::Request;
```

Comment:
Try `cargo fmt`, this  will merge the imports

Reply:
I did. It does not merge `use` statement. And the one `use` per line style reduces conflicts with `git-merge` and `git-rebase`.

Reply:
We don't use merge group feature, but the style is group, it's fine here